Wikipedia/Code/ColumnarCollectionViewController.swift

```diff
@@ -39,11 +39,15 @@ class ColumnarCollectionViewController: UICollectionViewController, Themeable {
                 cellWithSubItems.deselectSelectedSubItems(animated: animated)
             }
         }
+        NotificationCenter.default.addObserver(forName: .UIContentSizeCategoryDidChange, object: nil, queue: nil) {_ in
+            self.collectionView?.reloadData()
+        }
     }
     
     override func viewDidDisappear(_ animated: Bool) {
         super.viewDidDisappear(animated)
         unregisterForPreviewing()
+        NotificationCenter.default.removeObserver(self, name: .UIContentSizeCategoryDidChange, object: nil)
```

Comment:
can be removed after addressing comment above

Reply:
Fixed!